cvehound/cve/CVE-2021-3564.cocci

```diff
@@ -0,0 +1,27 @@
+/// Files: net/bluetooth/hci_core.c
+/// Fixes: 6a137caec23aeb9e036cdfd8a46dd8a366460e5d
```

Comment:
```
/// Fix: 6a137caec23aeb9e036cdfd8a46dd8a366460e5d
/// Detect-To: b78752cc71d86998d3b77d873c61d6ffdb7a2142
```

Reply:
b78752cc71d86998d3b77d873c61d6ffdb7a2142 seems like the commit where the bug appeared the first time

Reply:
If we added tasklet_call function into the rule shouldn't we change 'Detect-To' field?

Reply:
Commit b78752cc71d86998d3b77d873c61d6ffdb7a2142 contains tasklet_kill() function call.